src/dotnet/APIView/APIView/Token.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Microsoft.CodeAnalysis;
+
+namespace APIView
+{
+    public class Token
+    {
+        public string DisplayString { get; set; }
+        public bool IsNavigable { get; set; }
+        public bool IsString { get; set; }
+        public string NavigationID { get; set; }
+        public TypeReference.TypeName Type { get; set; }
+
+        public Token()
+        {
+            this.DisplayString = "";
+            this.IsNavigable = false;
+            this.IsString = false;
+            this.NavigationID = "";
+            this.Type = TypeReference.TypeName.NullType;
+        }
+
+        public Token(TypedConstant part)
```

Comment:
Hm, I wouldn't use Token to represent TypedConstant

Reply:
This actually just got tossed - was leftover from an attempt to tokenize attribute constructor arguments